src/app/core/utils/navigation-listener.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { Subject, Subscription } from 'rxjs';
+import { AlertService, HttpError, NavigationNotifierService, NavigationOrigin } from '@hmcts/ccd-case-ui-toolkit';
+
+@Injectable()
+export class NavigationListenerService {
+  public static readonly CASE_CREATED_MSG = 'The case has been created successfully';
+
+  callbackErrorsSubject: Subject<any> = new Subject();
+  navigationSubscription: Subscription;
+
+  constructor(
+    private alertService: AlertService,
+    private navigationNotifier: NavigationNotifierService,
+    private router: Router) {
+  }
+
+  public init() {
+    this.navigationSubscription = this.navigationNotifier.navigation.subscribe(navigation => {
+      switch (navigation.action) {
+        case NavigationOrigin.DRAFT_DELETED:
+          return this.router.navigate(['list/case'])
+            .then(() => {
+              this.alertService.setPreserveAlerts(true);
+              this.alertService.success(`The draft has been successfully deleted`);
+            });
+        case NavigationOrigin.ERROR_DELETING_DRAFT:
+          return this.router.navigate(['list/case']);
+        case NavigationOrigin.DRAFT_RESUMED:
+          return this.router.navigate(
+            ['create/case',
+              navigation.jid,
+              navigation.ctid,
+              navigation.etid], { queryParams: navigation.theQueryParams }).catch(error => {
+            this.handleError(error, navigation.etid);
+          });
+        case NavigationOrigin.EVENT_TRIGGERED:
+          return this.router.navigate(['trigger', navigation.etid], {
+            queryParams: navigation.theQueryParams,
+            relativeTo: navigation.relativeTo
+          }).catch(error => {
+            this.handleError(error, navigation.etid);
+          });
+        case NavigationOrigin.NO_READ_ACCESS_REDIRECTION:
+          return this.router.navigate((['/list/case']));
```

Comment:
We need to alert successful creation of case here.

Reply:
👍